src/connection/database/index.js

```diff
@@ -174,7 +174,7 @@ function processDatabaseOptions(opts) {
         prefill = undefined;
       }
 
-      const types = ['select', 'text', 'checkbox', 'hidden', 'textinfo'];
+      const types = ['select', 'text', 'checkbox', 'hidden', 'textinfo', 'radio_group'];
```

Comment:
to be consistent with other types: radio_group -> radiogroup

Reply:
done